### ISSUE TITLE
Custom Add to Cart Link Enhancements

### ISSUE THREAD

**Issue by sdavis2702** [easydigitaldownloads/Easy-Digital-Downloads#3063]
Here's a nice feature request: https://easydigitaldownloads.com/support/topic/custom-add-to-cart-urls-multiple-downloads/

General idea is to be able to control item quantities as well as adding multiple items to the cart with one link. Feasible?

**Comment by ghost1227** [comment]
It's feasible, but would take some reworking of the shortcode. I like the idea though!

**Comment by ScottDeLuzio** [comment]
Sean, thanks for starting the conversation for looking into adding this feature. I originally got the idea when working with another site that used OpenCart and had this extension installed: http://www.opencart.com/index.php?route=extension/extension/info&extension_id=11443
I know EDD and OpenCart are two completely separate softwares, but thought it might help to visualize what I was talking about.
Thanks again.

**Comment by sdavis2702** [comment]
@ghost1227 I hope not too much. I can definitely see this being useful, though.

@ScottDeLuzio No problem and thanks for the additional reference! Dan up there and a couple of the other developers will dig deeper most likely next week to discuss if it can/will happen or not, and how. So just stay tuned to this Issue and we'll see! :+1:

**Comment by pippinsplugins** [comment]
Setting the quantity is possible; multiple items are not (yet). I'd love to support it.

**Comment by dtxen1985** [comment]
functionality is already available ?
I like to be able to control item quantities as well as adding multiple items to the cart with one link.

**Comment by SDavisMedia** [comment]
@dtxen1985 No, the functionality is scheduled for version 2.7, though there's no guarantee that it will make the cut. We are currently on version 2.6.5. We do not have an exact timeline for this release.

**Comment by pippinsplugins** [comment]
Turns out this already works: `https://edd.com/?edd_action=add_to_cart&download_id=5072&edd_options[quantity]=2`

**Comment by mwhiteley16** [comment]
Is there a time-frame on possibly adding the ability to add multiple downloads to the cart using a single URL?

**Comment by pippinsplugins** [comment]
@mwhiteley16 not at this time.
